compiler/bcq-tools/generate_bcq_output_arrays

```diff
@@ -1,5 +1,20 @@
 #!/usr/bin/env python3
 
+# Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+# Copyright (C) 2018 The TensorFlow Authors
```

Comment:
does this code has direct copy from tensorflow?

Reply:
There is no direct code from Tensorflow but the codes import TensorFlow. So I thought that we should write the license author like `tf2tfliteV2.py`

Reply:
tf2tfliteV2.py has copied code from TensorFlow :)
If there is no direct copy code then we can skip this if you agree.

Reply:
Oh, I didn't know that @.@
Frankly speaking, I totally agree with you but I want to be checked that we are really correct.
So after it is checked, I will remove it :)